README.md

```diff
@@ -813,7 +815,7 @@ This applies to snippets and samples.
 
 #### Sometimes current patch
 
-In some cases, usually where a package has significant new API in a minor, it may be necessary to version snippets down to the "current patch".
+Due to DependencyVersion mode `HighestMinor` it is not needed to pin to a minor unless the referenced package is not following SemVer. It may be necessary to version snippets down to the "current patch".
```

Comment:
```suggestion
In some cases, usually where a package has significant new API in a minor, it may be necessary to version snippets down to the "current patch".
```